reconcile/gabi_authorized_users.py

```diff
@@ -10,6 +11,7 @@
 )
 from typing import (
     Any,
+    Callable,
```

Comment:
please import `Callable` from `collections.abc`

Reply:
Done in https://github.com/app-sre/qontract-reconcile/pull/3219/commits/bb4bf6d7599d02d7bcf7a6ab64bdf5e5f22aaf59. Thank you!